### ISSUE TITLE
User profile lookup failing

### ISSUE THREAD

**Issue by gregory-h** [dreadjr/passport-bitly#4]
It seems that Bitly has deprecated user profiles, so the Bitly strategy is returning 404 page not found on the user profile. 

Is there a way to disable profile lookup at the Passport api?

**Comment by dreadjr** [comment]
Appears v3 was deprecated.
https://support.bitly.com/hc/en-us/articles/360004395631-Migrating-from-V3-to-V4-of-the-Bitly-API

Looks like the v4 call is just /user vs /user/info

**Comment by dreadjr** [comment]
@gregory-h Can you test out the latest release?

**Comment by gregory-h** [comment]
Sure but having issues installing. I've updated my package.json with "passport-bitly": "0.1.3", but npm install complains 

npm install
npm ERR! code ETARGET
npm ERR! notarget No matching version found for passport-bitly@0.1.3.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 

Tried with ^0.1.3 and 0.1.3 but same result.

Is there something else I need to do?
